crates/api_common/src/post.rs

```diff
@@ -17,6 +17,7 @@ pub struct CreatePost {
   pub body: Option<String>,
   pub honeypot: Option<String>,
   pub nsfw: Option<bool>,
+  pub language: Option<String>,
```

Comment:
This can also be `language_id`, since you have these language_ids in the front end.